### ISSUE TITLE
Build frameworks for arm64 architecture as well ?

### ISSUE THREAD

**Issue by badrinathvm** [grab/cocoapods-binary-cache#68]
### Checklist

- [x] I've read the [Contribution Guidelines](https://github.com/grab/cocoapods-binary-cache/blob/master/CONTRIBUTING.md)
- [x] I've searched for [existing GitHub issues](https://github.com/grab/cocoapods-binary-cache/issues) if there was such a request before.

### Description
As of now when we use `pod binary prebuild`, it prebuilds  the framework line below 

`xcodebuild -project /Users/XXX/projects/XXX/XXX/_Prebuild/Pods.xcodeproj -target IntuitAuthorization -configuration Debug -sdk iphonesimulator -destination "generic/platform=iOS Simulator" ARCHS=x86_64 ONLY_ACTIVE_ARCH=NO build 2>&1`

How to can we include ARCHS=x86_64, arm64?  so that archiving works.

#### Motivation

<details><pre>[Details go here]</pre></details>

#### Summary

<details><pre>[Details go here]</pre></details>

**Comment by Oxthor** [comment]
This is related to this issue: https://github.com/grab/cocoapods-binary-cache/issues/19
And it should be fixed by adding a new option to the configuration: `device_build_enabled: true`

**Comment by badrinathvm** [comment]
Thank you @Oxthor , Let me give a try and keep posted.

**Comment by badrinathvm** [comment]
So to clarify more, after making ` dev_pods_enabled: true` i should be seeing `ARCHS=x86_64,arm64` and -sdk as `iPhoneos`

`xcodebuild -project /Users/XXX/projects/XXX/XXX/_Prebuild/Pods.xcodeproj -target IntuitAuthorization -configuration Debug -sdk iphonesimulator -destination "generic/platform=iOS Simulator" **ARCHS=x86_64,arm64** ONLY_ACTIVE_ARCH=NO build 2>&1` ?

But i am still seeing like below.

![Screen Shot 2021-02-17 at 1 49 30 PM](https://user-images.githubusercontent.com/1479179/108272353-f8b15800-7126-11eb-84b5-1f02fc0973a5.png)

**Comment by Oxthor** [comment]
It is a two step build, first simulator, then devices. Wait a bit more and once the simulator one finishes, you will see an extra xcodebuild command running for device.

Just to clarify, this is the params you should be setting: `device_build_enabled: true`.

**Comment by badrinathvm** [comment]
@Oxthor Let me try this quickly and get back to u for any other issues i see.

**Comment by badrinathvm** [comment]
Yes, it is executing one more command after a delay. Thank you. I have another question if I use `pod binary prebuild --push` is` --push` is considered as optional only to push if any delta exists or pushes every time ? Looks like it is executing every time and build fails ( See Below ) How to make this push optional ?
```nothing to commit, working tree clean

――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

### Command

```
/Users/****/.rvm/gems/ruby-2.5.0/bin/pod binary prebuild --repo-update --push
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?


### Stack

```
   CocoaPods : 1.8.4```
